app/subscriber/src/components/tool-bar/ContentListActionBar.tsx

```diff
@@ -0,0 +1,5 @@
+import { ContentActionBar, IContentActionBarProps } from './ContentActionBar';
+
+export const ContentListActionBar: React.FC<IContentActionBarProps> = ({ className, ...rest }) => {
```

Comment:
Since a common implementation is to add the className, we can create a simple wrapper.